app/controllers/catalog_controller.rb

```diff
@@ -81,7 +74,7 @@ def self.modified_field
       next if Document.properties[prop].behaviors.nil? || Generic.properties.key?(prop)
 
       # Add property as searchable all fields box and individually
-      if Document.properties[prop].behaviors.include?(:stored_searchable)
+      next unless Document.properties[prop].behaviors.include?(:stored_searchable)
```

Comment:
Take note of this change. This used to be an if but it wants a next. Since adding the next, it would skip it if the statement were true. This change SHOULD fix that. More testing is needed though

Reply:
Strike that. I need to dig into this further. But something has fundamentally changed about this statement that I need to discover.